### ISSUE TITLE
Internet permission

### ISSUE THREAD

**Issue by davidmef** [Lucchetto/SuperImage#48]
I see in https://f-droid.org/en/packages/com.zhenxiang.superimage/ that version 1.3.4 added permissions:
- have full network access
- view network connections
- view Wi-Fi connections

What does it need these permissions for?

**Comment by marek22k** [comment]
I was also just about to open an issue with the same question. I would be interested in that as well.

**Comment by solofireflyxdev** [comment]
Hello sir 

"why it's asking you for the full access and all those stuffs for the permission says it to the why it's uses Wi-Fi if this doesn't be unable to your app will not allowed to be used anything with Wi-Fi on your app"

Kind regards,
CEO & founder at toolswebdev

**Comment by VasilisKos** [comment]
I've noticed that too, and it probably has to do with the addition of the Google Ads add-on!
Are you planning to add ads to the app? 🤨

P. S. Also, what's the point of adding a notification and a link to the PC version in the Android app?

**Comment by Wjxfi** [comment]
![2fa044a2-a688-4dcd-93d1-3dabb840e983.jpg](https://github.com/Lucchetto/SuperImage/assets/113500363/7d4b848e-d6b9-4af1-9923-d1f423d4d3eb)

**Comment by solofireflyxdev** [comment]
> I've noticed that too, and it probably has to do with the addition of the Google Ads add-on!
> Are you planning to add ads to the app? 🤨
> 
> P. S. Also, what's the point of adding a notification and a link to the PC version in the Android app? 

Note: The app needs wi-fi to work

Kind regards 
CEO & founder of toolswebdev, inc™

**Comment by VasilisKos** [comment]
> > I've noticed that too, and it probably has to do with the addition of the Google Ads add-on!
> > Are you planning to add ads to the app? 🤨
> > P. S. Also, what's the point of adding a notification and a link to the PC version in the Android app?
> 
> Note: The app needs wi-fi to work
> 
> Kind regards CEO & founder of toolswebdev, inc™

Say what? What for? 🤨

**Comment by solofireflyxdev** [comment]
> > > I've noticed that too, and it probably has to do with the addition of the Google Ads add-on!
> > > Are you planning to add ads to the app? 🤨
> > > P. S. Also, what's the point of adding a notification and a link to the PC version in the Android app?
> > 
> > Note: The app needs wi-fi to work
> > 
> > Kind regards CEO & founder of toolswebdev, inc™
> 
> Say what? What for? 🤨

Note: if you want an app to work you'll have to give the device full access to the internet connection then where is it getting internet from Bluetooth or something?

Does permissions are for the app to function properly 

Kind regards 
CEO & founder of toolswebdev, inc™

**Comment by Lucchetto** [comment]
Hi, I've added matomo to have a overview of the f-droid userbase. After looking at the numbers I think I'll be dropping the f-droid version

**Comment by solofireflyxdev** [comment]
> Hi, I've added matomo to have a overview of the f-droid userbase. After looking at the numbers I think I'll be dropping the f-droid version

Ok 

Kind regards
CEO & fonder of toolswebdev

**Comment by VasilisKos** [comment]
> Hi, I've added matomo to have a overview of the f-droid userbase. After looking at the numbers I think I'll be dropping the f-droid version

What about the Google Ads?

**Comment by Lucchetto** [comment]
> > Hi, I've added matomo to have a overview of the f-droid userbase. After looking at the numbers I think I'll be dropping the f-droid version
> 
> What about the Google Ads?

Yes play store version has ads now, I'll add an option to remove ads soon

**Comment by VasilisKos** [comment]
> > > Hi, I've added matomo to have a overview of the f-droid userbase. After looking at the numbers I think I'll be dropping the f-droid version
> > 
> > 
> > What about the Google Ads?
> 
> Yes play store version has ads now, I'll add an option to remove ads soon

Through payment? 🤨

**Comment by Lucchetto** [comment]
Yes

**Comment by ivan-avalos** [comment]
I suspect that most of us who are using SuperImage chose it over the hundreds of shady AI apps in the Play Store for privacy reasons, because we wanted an open-source app that did its processing entirely client-side. Selling your userbase to Google will only help alienate them, and for the small amount of money that Google will give you it's not even worth it: we're not worth that.

How about making your app paid-only in the Play Store instead? There are many successful open-source apps doing this, such as Conversations, Threema, and D-Sub.

**Comment by RobbWatershed** [comment]
Guys, do you realize you're agonizing over an open-source, forkable codebase? You can make your own flavour if you're unhappy with the owner's choices.
